docs/admin/deployment.md

```diff
@@ -452,14 +452,19 @@ storage selection.
 
 #### Discovery
 
-`dmg storage scan` can be run to query remote running `daos_server`
-processes over the management network.
-
-`daos_server storage scan` can be used to query local `daos_server` instances
-directly (scans locally-attached SSDs and Intel Persistent Memory Modules usable
-by DAOS).
-NVMe SSDs need to be made accessible first by running
-`daos_server storage prepare --nvme-only`.
+DAOS tools will discover NVMe SSDs and Persistent Memory Modules using the storage scan commands.
+
+`dmg storage scan` can be run to query remote running `daos_server` processes over the management
+network.
+
+`daos_server storage scan` can be used to query storage on local hosts directly.
```

Comment:
on the local host (singular)

Reply:
Note that daos_server should not be running (e.g. as a systemd service under the daos_server userid) when the `daos_server storage scan` is executed, as the NVMe SSDs will then already be bound to the daos_server processes, and trying to access them (as a non-daos_server user, even as root) will cause access failures.